packages/db/index.ts

```diff
@@ -1,5 +1,22 @@
 import { PrismaClient } from "@prisma/client";
 
-export const prisma = new PrismaClient({
-  log: ["query"],
-});
+declare global {
+  // allow global `var` declarations
+  // eslint-disable-next-line no-var
+  var prisma: PrismaClient | undefined;
+}
+
+export const prisma =
+  global.prisma ||
+  new PrismaClient({
+    log:
+      process.env.NODE_ENV === "development"
+        ? ["query", "error", "warn"]
+        : ["error"],
+  });
+
+if (process.env.NODE_ENV !== "production") {
```

Comment:
this makes it so our prisma client doesn't duplicate itself and end up giving warnings that we have instantiated 10+ clients

Reply:
Great. 👍